src/debug.py

```diff
@@ -4,23 +4,26 @@
 =============================
 
 Levels:
-    DEBUG       Detailed information, typically of interest only when
-                diagnosing problems.
-    INFO        Confirmation that things are working as expected.
-    WARNING     An indication that something unexpected happened, or indicative
-                of some problem in the near future (e.g. ‘disk space low’).
-                The software is still working as expected.
-    ERROR       Due to a more serious problem, the software has not been able
-                to perform some function.
-    CRITICAL    A serious error, indicating that the program itself may be
-                unable to continue running.
+
+   DEBUG
+      Detailed information, typically of interest only when diagnosing problems.
+   INFO
+      Confirmation that things are working as expected.
+   WARNING
+      An indication that something unexpected happened, or indicative of some problem in the
+      near future (e.g. ‘disk space low’). The software is still working as expected.
+   ERROR
+      Due to a more serious problem, the software has not been able to perform some function.
+   CRITICAL
+      A serious error, indicating that the program itself may be unable to continue running.
 
 There are three loggers: `console_only`, `file_only` and `both`.
 
-Use: `from debug import logger` to import this facility into whatever module
-     you wish to log messages from. Logging is thread-safe so you don't have
-     to worry about locks, just import and log.
+Use: `from debug import logger` to import this facility into whatever module you wish to log messages from.
+    Logging is thread-safe so you don't have to worry about locks, just import and log.
+
 """
+
 import logging
```

Comment:
if possible can we place imports at the starting of the file.

Reply:
Are you worried about the blank line or the module docstring?